src/dataset-builder/ConceptSearch.ts

```diff
@@ -109,15 +108,19 @@ export const ConceptSearch = (props: ConceptSearchProps) => {
               { width: 100, key: 'hierarchy' },
             ],
             rows: _.map((concept) => {
-              const [label, iconName]: [string, IconId] = _.contains(concept, cart)
-                ? ['remove', 'minus-circle-red']
-                : ['add', 'plus-circle-filled'];
+              const checked = _.contains(concept, cart);
               return {
                 name: div({ style: { display: 'flex' } }, [
-                  h(Link, { 'aria-label': `${label} ${concept.id}`, onClick: () => setCart(_.xor(cart, [concept])) }, [
-                    icon(iconName, { size: 16 }),
-                  ]),
-                  div({ style: { marginLeft: 5 } }, [
+                  h(
+                    LabeledCheckbox,
+                    {
+                      'aria-label': checked ? `uncheck ${concept.id}` : `check ${concept.id}`,
+                      checked,
+                      onChange: () => setCart(_.xor(cart, [concept])),
+                    },
+                    []
```

Comment:
If this is a labeled check box, shouldn't the name now be the check box label? It seems to me that it may be surprising for a user that the name isn't part of the checkbox control.

Reply:
Can you explain on this more?